From f67273561152231c6f3022eef576e558626d126c Mon Sep 17 00:00:00 2001
Message-Id: <f67273561152231c6f3022eef576e558626d126c.1727074050.git.mykyta_poturai@epam.com>
From: Mykyta Poturai <mykyta_poturai@epam.com>
Date: Mon, 23 Sep 2024 09:45:21 +0300
Subject: [PATCH] arm: Change GUEST_GICV3_ITS_BASE

For PCI passthrough to work correctly with MSI interrupts guest's ITS
base needs to match with physical ITS address.

Signed-off-by: Mykyta Poturai <mykyta_poturai@epam.com>
---
 xen/include/public/arch-arm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xen/include/public/arch-arm.h b/xen/include/public/arch-arm.h
index 89b986f89d..7063f96d62 100644
--- a/xen/include/public/arch-arm.h
+++ b/xen/include/public/arch-arm.h
@@ -430,7 +430,7 @@ typedef uint64_t xen_callback_t;
 #define GUEST_VPCI_ECAM_SIZE    xen_mk_ullong(0x10000000)
 
 /* vGIC ITS mappings */
-#define GUEST_GICV3_ITS_BASE   xen_mk_ullong(0x04020000)
+#define GUEST_GICV3_ITS_BASE   xen_mk_ullong(0xf1040000)
 #define GUEST_GICV3_ITS_SIZE   xen_mk_ullong(0x00020000)
 
 /* ACPI tables physical address */
-- 
2.34.1

